js/classes.js

```diff
@@ -1,121 +1,123 @@
+    this.name = name;
+    this.type = type;
 }
 
-Ingredient.prototype.getName = function () {
-	return this.name;
+Ingredient.prototype.getName = function() {
+    return this.name;
 };
 
-Ingredient.prototype.getType = function () {
-	return this.type;
+Ingredient.prototype.getType = function() {
+    return this.type;
 };
 
 // Drink Object
-var Drink = function () {
-	this.ingredient = [];
+var Drink = function() {
+    this.ingredient = [];
 }
 
-Drink.prototype.addIngredient = function (ingredient) {
-	this.ingredient.push(ingredient);
+Drink.prototype.addIngredient = function(ingredient) {
+    this.ingredient.push(ingredient);
 };
 
-Drink.prototype.getDrinkType = function () {
-	var drinkType = 'yer drink is ';
-	for (var i = 0; i < this.ingredient.length; i++) {
-		drinkType += this.ingredient[i].type + ' ';
-	}
+Drink.prototype.getDrinkType = function() {
+    var drinkType = 'yer drink is ';
+    for (var i = 0; i < this.ingredient.length; i++) {
+        drinkType += this.ingredient[i].type + ' ';
+    }
 };
 
-Drink.prototype.getDrinkIngredients = function () {
-	var drinkIngredients = 'yer drink has ';
-	for (var i = 0; i < this.ingredient.length; i++) {
-		drinkIngredients += this.ingredient[i].name + ' ';
-	}
+Drink.prototype.getDrinkIngredients = function() {
+    var drinkIngredients = 'yer drink has ';
+    for (var i = 0; i < this.ingredient.length; i++) {
+        drinkIngredients += this.ingredient[i].name + ' ';
+    }
 };
 
 // Pantry Object
-var Pantry = function () {
-	this.ingredients = {
-		'salty' : ['Olive on a stick', 'salt-dusted rim', 'rasher of bacon'],
-		'strong' : ['Glug of rum', 'slug of whisky', 'splash of gin'],
-		'sweet' :  ['Sugar cube', 'spoonful of honey', 'splash of cola'],
-		'bitter' : ['Shake of bitters', 'splash of tonic', 'twist of lemon peel'],
-		'fruity' : ['Slice of orange', 'dash of cassis', 'cherry on top'],
- 	};
+var Pantry = function() {
+    this.ingredients = {
+        'salty': ['Olive on a stick', 'salt-dusted rim', 'rasher of bacon'],
+        'strong': ['Glug of rum', 'slug of whisky', 'splash of gin'],
+        'sweet': ['Sugar cube', 'spoonful of honey', 'splash of cola'],
+        'bitter': ['Shake of bitters', 'splash of tonic', 'twist of lemon peel'],
+        'fruity': ['Slice of orange', 'dash of cassis', 'cherry on top'],
+    };
 
 };
 
-Pantry.prototype.getIngredient = function (type) {
-	var numb = Math.floor(Math.random()*5)
-	console.log(type, "some type");
-	var ingredient = this.ingredients[type][numb]; /* this.ingredients[type].length-1)] */
-	if (ingredient != undefined )
-		return ingredient;
+Pantry.prototype.getIngredient = function(type) {
+    var numb = Math.floor(Math.random() * 5)
+    console.log(type, "some type");
+    if (this.ingredients[type] !== undefined) {
+        var ingredient = this.ingredients[type][numb]; /* this.ingredients[type].length-1)] */
+        if (ingredient != undefined) {
+            return ingredient;
+        }
+    }
```

Comment:
Please focus on the `getIngredient` method. 
The other changes are because of a code prettify plugin.